compiler+runtime/include/cpp/jank/type.hpp

```diff
@@ -33,7 +33,8 @@ namespace jank
   using native_big_integer
     = boost::multiprecision::number<boost::multiprecision::cpp_int_backend<>>;
   using native_big_decimal
-    = boost::multiprecision::number<boost::multiprecision::cpp_dec_float<100>>;
+    = boost::multiprecision::number<boost::multiprecision::cpp_dec_float<100>,
```

Comment:
I wasn't sure exactly how to address this analyzer warning, and this was the only thing that made sense that I could find online.

Reply:
Hm, what was the warning? I'm not sure what the problem was and what you changed.

Reply:
Hm, which `abs` fn is being called there? I'm not finding the `abs` which takes a `native_big_decimal`.

Reply:
I tried the above, and still get the same analyzer warning. The only thing I needed to fix the issues is adding `boost::multiprecision::et_off`.

Reply:
Ok, gotcha. I read up on the docs for that. I think we can keep your change as is.